demo/app.js

```diff
@@ -15,6 +15,7 @@ class App extends React.Component {
       underlineHeader: false,
       zoomScale: 0.5,
       slidesToShow: 1,
+      slidesToScroll: 1,
```

Comment:
Added this to state in our demo app so we can easier update this param when testing.